### PR TITLE
net-misc/i2pd: 2.9.0 version bump

### PR DESCRIPTION
**Pull request by khumarahn**
Hi, this is another version bump. I did not make any changes to the 2.8.0. ebuild.
@blueness

### CODE REVIEW

File: net-misc/i2pd/i2pd-2.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	|| ( >=sys-devel/gcc-4.7 >=sys-devel/clang-3.3 )"
+
+I2PD_USER=i2pd
+I2PD_GROUP=i2pd
+
+CMAKE_USE_DIR="${S}/build"
+
+DOCS=( README.md docs/i2pd.conf debian/tunnels.conf debian/subscriptions.txt )
+
+src_prepare() {
```
Comment (doughdemon) [inline net-misc/i2pd/i2pd-2.9.0.ebuild]: This entire phase could be replaced by usage of the PATCHES array.

**Comment by khumarahn** [comment]
@doughdemon: I agree, changed it.

**Comment by khumarahn** [comment]
I forgot to cc other maintainers, I am sorry.
@klondi @tomboy-64

**Comment by blueness** [comment]
pushed

**Pull request closed without merging**